src/library/Uno.Themes.WinUI.Markup/Theme/HyperlinkButton.cs

```diff
@@ -0,0 +1,105 @@
+﻿using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Media;
+using Uno.Extensions.Markup.Internals;
+
+namespace Uno.Themes.Markup
+{
+	public static partial class Theme
+	{
+		public static class HyperlinkButton
+		{
+			public static class Resources
+			{
+				public static class Default
```

Comment:
I don't think we need this class at this level. It should just be HyperlinkButton.Resources.Foreground.Default/PointerOver/Pressed/etc.

Reply:
removed